app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -42,25 +42,69 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  swagger_schema :GetHighlights do
+    key :required, [:source_type]
+    property :page do
+      key :type, :integer
+      key :description, 'The page number'
+    end
+    property :per_page do
+      key :type, :integer
+      key :description, 'The number of highlights to return per page'
+    end
+    property :order do
+      key :type, :string
+      key :enum, %w[asc desc]
+      key :description, 'The returned sorted order'
+    end
+  end
+
   swagger_schema :Highlight do
     key :required, [:id]
   end
 
+  swagger_schema :Highlights do
+    property :highlights do
+      key :type, :array
+      key :description, 'One or more highlights'
+      items do
+        key :'$ref', :Highlight
+      end
+    end
+  end
+
   swagger_schema :NewHighlight do
     key :required, [:source_type, :source_id, :anchor, :highlighted_content, :color, :location_strategies]
   end
 
+  add_properties(:GetHighlights, :NewHighlight, :Highlight) do
+    property :source_type do
+      key :type, :string
+      key :enum, ['openstax_page']
+      key :description, 'The source_type of the highlight, typically a openstax_page'
```

Comment:
Probably shouldn't use "source_type" in the description of "source_type".  Maybe just "The type of content that contains the highlight, typically..."